### PR TITLE
Add projector.control_view action and tests.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: openslides_backend/action/actions/projector/control_view.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        },
+        additional_optional_fields={
+            "step": {"type": "integer"},
+        },
+    )
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        field = instance.pop("field")
+        direction = instance.pop("direction")
+        step = instance.pop("step", 1)
```
Comment (FinnStutzenstein) [inline openslides_backend/action/actions/projector/control_view.py:30]: I missed one thing in the docs: if `step < 1`, step is set to 1.

File: openslides_backend/action/actions/projector/control_view.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        elif direction == "up":
+            projector = self.datastore.get(
+                FullQualifiedId(self.model.collection, instance["id"]), [field]
+            )
+            new_value = projector.get(field, 0) + step
+        elif direction == "down":
+            projector = self.datastore.get(
+                FullQualifiedId(self.model.collection, instance["id"]), [field]
+            )
+            new_value = projector.get(field, 0) - step
```
Comment (FinnStutzenstein) [inline openslides_backend/action/actions/projector/control_view.py:43]: put some kind of `BadProgrammingError` or `LogicError` in the else part - this cannot be called, but if it is, we have an error somewhere.

**Review by FinnStutzenstein — APPROVED** [review]
Looks good, minor changes needed.

File: openslides_backend/action/actions/projector/control_view.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/projector/control_view.py:31]: I catch the `step < 1` and add a test for it. A step < 1 is an error.

File: openslides_backend/action/actions/projector/control_view.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/projector/control_view.py:44]: Add an exception to the else case.

**Review by FinnStutzenstein — APPROVED** [review]

**Pull request merged**